docs/plugins/composable_resources.md

```diff
@@ -276,6 +271,33 @@ acPostProcForDataCopyReceived(*leaf_resource) {
 }
 ```
 
+#### Detached Mode Configuration
+
+Detached mode was added as a configuration option for UnixFileSystem resources in release 4.3.1.  When detached mode is enabled, any server may serve up requests for the resource as long as all servers share a common mounted vault.
+
+To enable detached mode, the **host_mode** setting in the context string should be set to **detached** as in the following example:
+
+```
+irods@hostname:~/ $ iadmin mkresc detached_resc unixfilesystem hostname.example.org:/common/mount/point "host_mode=detached"
+```
+
+If host_mode is not set to "detached", the unixfilesystem plugin will default to attached mode.
```

Comment:
Probably should remove the double quotes since they aren't used in other places for this documentation.

Reply:
It was my thinking that the context is a bit different here.  I wanted to emphasize that it should be exactly this.  That said the check is not case sensitive.

Reply:
I went ahead and bolded host_mode and detached to emphasize these are exact settings.  I also add "(not case sensitive)" after detached.